matRad_shift.m

```diff
@@ -0,0 +1,81 @@
+function [projCoord,idx,targetPoint, sourcePoint] = matRad_shift(initIx,dim,sourcePoint_bev,targetPoint_bev,isoCenter, res, Dx, Dz, rotMat)
```

Comment:
1. IMO the name of the function is not instructive. Do you have a better suggestion?
2. as far as I understand you now make the projection for all components of one ray simultaneously to save time... this should be reflected in the function header making clear what is what!